Aviral Tyagi/problem4.hs

```diff
@@ -0,0 +1,9 @@
+myLength :: [a] -> Int
+myLength [] = 0
+myLength (_ : rest) = 1 + myLength rest 
+
```

Comment:
You can very easily add one more implementation of length. Check https://hoogle.haskell.org/ for the same type signature.